packages/api-docs/src/PageJsonGenerator.types.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Options that define the information necessary to find the api.json file
+ * and create individual page json files out of it
+ */
+export interface IPageJsonOptions {
+  apiJsonPaths: string[];
+  pageJsonFolderPath: string;
+  pageNames: string[];
+  kind: PageKind;
+}
+
+/**
+ * The name of the page and what kind of page it is
+ */
+export interface IPage {
+  pageName: string;
+  kind: PageKind;
+}
+
+/**
+ * Used to keep track of where the page will live on the site
+ */
+export enum PageKind {
+  References = 'References',
+  Components = 'Components'
+}
+
+export interface ITokenJson {
+  text: string;
+  hyperlinkedPage?: string;
+  pageKind?: PageKind;
+}
+
+export interface ITableRowJson {
+  name: string;
+  kind?: 'Method' | 'Property';
+  typeTokens: ITokenJson[];
+  defaultValue?: string;
+  descriptionHtml: string;
+  deprecated: boolean;
+  deprecatedMessage?: string;
+}
+
+export interface IEnumTableRowJson {
+  name: string;
+  descriptionHtml: string;
```

Comment:
Is the HTML text escaped? If so might be worth mentioning as part of the name.

Reply:
Or explaining the expected format in a comment :)

Reply:
Thanks for calling this out. Originally, I named this property `descriptionHtml` to handle rendering descriptions as html, but I don't think it's really necessary at the moment. I just renamed it back to `description` and we can revisit it later if we think that's something we want to support.